test/parse-lib

```diff
@@ -38,7 +38,7 @@ then
 elif [[ "$mode" == "wasm" ]]
 then
   # Ensure tree-sitter-haskell.wasm was compiled
-  make tree-sitter-haskell.wasm -s
+  tree-sitter build-wasm
```

Comment:
it's `tree-sitter build --wasm -o ...` now.

Reply:
thanks. 0.22 doesn't appear to be available from npm or nixpkgs yet though